lib/recaptcha/token.rb

```diff
@@ -0,0 +1,19 @@
+module Recaptcha
+  module Token
+    require 'json'
+
+    def self.secure_token
```

Comment:
yes, since it is it's own public api ... something simple is good enough ... just want to have a unit test so people know where to add if they change/fix it